setup.py

```diff
@@ -2,7 +2,7 @@
 
 def parse_requirements():
     with open('requirements.txt', 'r') as f:
-        return [r for r in f.read().splitlines() if not r.startswith('git') and not r.startswith('.')]
+        return ['{2} @ {0}'.format(*r.partition('#egg=')) if '#egg=' in r else r for r in f.read().splitlines()]
```

Comment:
Do we need to include remote packages in `packages`?

This has (evidently) always worked for us, but [the distutils documentation](https://docs.python.org/3.5/distutils/setupscript.html#listing-whole-packages) shows in-package specs only.

We could try using [`setuptool`'s `find_packages`](https://setuptools.readthedocs.io/en/latest/setuptools.html#using-find-packages) if you are up for it.

Reply:
```
      install_requires=parse_requirements(),
      include_package_data=True,
      packages=find_packages('src'),
```

We are using find_packages. 
The parse_requirements is for `install_requires` so `pip install .` works.